.github/workflows/software.yml

```diff
@@ -0,0 +1,41 @@
+name: build software
+on:
+  schedule:
+    - cron: "0 0 * * 1"
+  push:
+    branches:
+      - master
+    paths:
+      - 'software/**'
+
+jobs:
+  cassandra:
+    name: "software:${{ matrix.cassandra }}"
```

Comment:
```suggestion
    name: "software:${{ matrix.tag }}"
```

Reply:
Solved by replacing the workflow with a common yolks workflow and adjusting it to fit the Cassandra tags and paths.